src/stdatamodels/validate.py

```diff
@@ -12,10 +12,7 @@
 import numpy as np
 
 from .util import convert_fitsrec_to_array_in_tree, remove_none_from_tree
-
-
-class ValidationWarning(Warning):
-    pass
+from stdatamodels.exceptions import ValidationWarning
```

Comment:
Thanks for opening https://github.com/spacetelescope/jwst/pull/9280 (adding this comment to xref that the import here is currently used by jwst but won't be once the linked PR is merged).